Packages/Tracking/Physical Hands/Runtime/Scripts/ContactBone.cs

```diff
@@ -223,13 +223,21 @@ private void UpdateObjectDistances(Collider[] colliderCache, int count)
 
                 if (IsPalm)
                 {
-                    // Do palm logic
-                    colliderPos = ContactUtils.ClosestPointEstimationFromRectangle(_palmColCenter, _palmColRotation, _palmExtentsReduced.xz(), collider);
-                    bonePos = ContactUtils.ClosestPointToRectangleFace(_palmPoints, _palmPlane.ClosestPointOnPlane(colliderTransformPosition));
-                    midPoint = colliderPos + (bonePos + ((bonePos - colliderPos).normalized * width) - colliderPos) / 2f;
+                    if (collider is BoxCollider || collider is SphereCollider || collider is CapsuleCollider || (collider is MeshCollider meshCollider && meshCollider.convex))
```

Comment:
To reduce the number of type checks here, could you invert this logic to just check `if (collider is MeshCollider meshCollider && !meshCollider.convex)`?